src/prepack-cli.js

```diff
@@ -225,13 +256,26 @@ function run(
         default:
           if (arg in flags) {
             flags[arg] = true;
+            reproArguments.push("--" + arg);
           } else {
             console.error(`Unknown option: ${arg}`);
             process.exit(1);
           }
       }
     }
   }
+
+  if (reproFilePath !== undefined) {
+    const zip = zipFactory();
+    for (let fileName of reproFileNames) {
+      let content = fs.readFileSync(fileName, "utf8");
+      zip.file(path.basename(fileName), content);
+    }
+    zip.file("repro.sh", "prepack " + reproArguments.map(a => `"${a}"`).join(" "));
```

Comment:
On my laptop this invokes /usr/local/bin/prepack which is defined as
#!/usr/bin/env node

require("../lib/prepack-cli");

That version of prepack behaves differently from the version in "./lib/prepac-cli.js"